pkg/util/cephcmds.go

```diff
@@ -69,14 +68,25 @@ func GetPoolID(monitors string, cr *Credentials, poolName string) (int64, error)
 		"osd", "lspools")
 	if err != nil {
 		klog.Errorf("failed getting pool list from cluster (%s)", err)
-		return 0, err
+		return nil, err
 	}
 
 	var pools []cephStoragePoolSummary
 	err = json.Unmarshal(stdout, &pools)
 	if err != nil {
 		klog.Errorf("failed to parse JSON output of pool list from cluster (%s)", err)
-		return 0, fmt.Errorf("unmarshal failed: %+v.  raw buffer response: %s", err, string(stdout))
+		return nil, fmt.Errorf("unmarshal failed: %+v.  raw buffer response: %s", err, string(stdout))
```

Comment:
@ShyamsundarR may be "unmarshal of pool list failed..." looks more informative.

Reply:
Done.